ezcater_rubocop.gemspec

```diff
@@ -25,8 +25,8 @@ Gem::Specification.new do |spec|
   spec.files = `git ls-files -z`.split("\x0").reject do |f|
     f.match(%r{^(test|spec|features)/})
   end
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir = "bin"
```

Comment:
aversions to keeping project-specific binstubs and gem inclusion-specific binstubs separate?

Reply:
oh wait is this so that we can also use the `circle_rubocop.rb` in this project?

Reply:
I changed the `bindir` because it seemed to match the most recent recommendations for structuring gems. I was not actually planning to run this script for this gem's circle build.

I'd be okay with separating the contents of the current "bin" directory, but I'm not sure it is necessary.

The `setup` and `console` files do get packaged with the gem currently, but only the `circle_rubocop.rb` script is executable, and therefore easily executed from the installed gem.

Rather than splitting, the better approach may be to exclude the other "bin" files from being packaged?

Reply:
makes sense 👍 TIL that that's the direction the community is taking. i think this approach makes more sense anyway, at the end of the day they're all binstubs only expose the ones you want, rite?